pytest_html/extras.py

```diff
@@ -8,17 +8,33 @@
 FORMAT_TEXT = 'text'
 FORMAT_URL = 'url'
 
+TYPE_JPG = ('image/jpeg', 'jpg')
+TYPE_PNG = ('image/png', 'png')
+TYPE_SVG = ('image/svg+xml', 'svg')
 
-def extra(content, format, name=None):
-    return {'name': name, 'format': format, 'content': content}
+
+def extra(content, format, name=None, type=None):
+    return {'name': name, 'format': format, 'content': content, 'type': type}
 
 
 def html(content):
     return extra(content, FORMAT_HTML)
 
 
 def image(content, name='Image'):
-    return extra(content, FORMAT_IMAGE, name)
+    return imagePNG(content=content, name=name)
+
+
+def imagePNG(content, name='Image'):
+    return extra(content, FORMAT_IMAGE, name, type=TYPE_PNG)
+
+
+def imageJPG(content, name='Image'):
+    return extra(content, FORMAT_IMAGE, name, type=TYPE_JPG)
+
+
+def imageSVG(content, name='Image'):
+    return extra(content, FORMAT_IMAGE, name, type=TYPE_SVG)
```

Comment:
I'm undecided on these helper functions. I think allowing users the specify the mime type and extension gives them full flexibility without too much complexity, but I can also see the advantage of them.